packaging/Dockerfile.ubuntu.20.04

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2019-2021, Intel Corporation
```

Comment:
@JohnMalmberg These copyright dates are from the packaging repo.    This will need to be corrected with a PR to packaging.    Should I do that? it will affect all rpm repos correct?

Reply:
No, that was my Bad.   Copyright dates on the packaging repo should only be updated when the packaging repo content is changed, not when another project copies them.